cli/compose/loader/loader.go

```diff
@@ -362,11 +362,11 @@ func formatInvalidKeyError(keyPrefix string, key interface{}) error {
 
 // LoadServices produces a ServiceConfig map from a compose file Dict
 // the servicesDict is not validated if directly used. Use Load() to enable validation
-func LoadServices(servicesDict map[string]interface{}, workingDir string, lookupEnv template.Mapping) ([]types.ServiceConfig, error) {
+func LoadServices(servicesDict map[string]interface{}, workingDir, homeDir string, lookupEnv template.Mapping) ([]types.ServiceConfig, error) {
```

Comment:
Looking at that function, we should probably create a `LoadServicesFromConfigDetails` (which would take `configDetails` as argument instead of those 3) or something.
That said `LoadServices` is exported so… if we break it once, let's try to break it in a future-proof way

Reply:
Should I add this function alongside the existing one or rename `LoadServices` to  `LoadServicesFromConfigDetails`  (and modify its parameters) ?

Reply:
Hum actually, looking back, I'm not sure anymore :sweat_smile: